Sources/Crypto/Util/SecureBytes.swift

```diff
@@ -62,7 +62,12 @@ extension SecureBytes {
     @inlinable
     mutating func append<C: Collection>(_ data: C) where C.Element == UInt8 {
         let requiredCapacity = self.count + data.count
-        if !isKnownUniquelyReferenced(&self.backing) || requiredCapacity > self.backing.capacity {
+#if os(OpenBSD)
+        let backingCapacity = self.backing.header.capacity
+#else
+        let backingCapacity = self.backing.capacity
```

Comment:
Is there any reason we can't offer a computed property in the OpenBSD case that can offer the `capacity`? That would minimise the scope of this diff, which would help.

Reply:
I originally did this because I was slightly concerned about the slight performance impact, but I've added a new commit that adds a computed property (so in the worst case, that commit can be reverted on its own without breaking swift-crypto for the platform). 

This necessitates redefining `_copyBytes` and `_withVeryUnsafeMutableBytes` as `@usableFromInline` instead of `@inlinable` on all platforms.